release/release.go

```diff
@@ -28,26 +31,32 @@ type release struct {
 func NewRelease(
 	name string,
 	version string,
+	description string,
```

Comment:
should we move all the strings into a mini struct ReleaseMeta so that it's easier to discern? positional args with same type really end up confusing.

Reply:
@cppforlife So I was about to make the requested changes, but after looking into it, I'm now wondering how useful they are. Here's why:
- `release.NewRelease` is the only candidate which uses positional arguments. All other uses are structs and therefore named. Agree that `release/release_test.go` is a bit ugly with all the `""`s, but don't thing it's too bad, because it's restricted mainly to this file.
- Other places like `release.Release` become a bit weird,  because the way name, version and description are treated is not the equivalent (description coming from the config, name and version coming from CLI args, see [`fs_release_dir.go`]()). That would require some weird constructs to make it work.
- `manifest.Manifest` models the yaml version of a manifest and I think it makes sense to maintain exactly the same structure as in the yaml file itself instead of using more nested levels in the Go struct than in the yaml.
- `director.Release` has yet again a slightly different structure: it contains an additional `VersionMark` function, and `Version` is of type `SemVer`, not `string`.

So, overall I don't think introducing a `ReleaseMeta` struct would actually help to make the code easier to read and maintain. What do you think?